.vscode/settings.json

```diff
@@ -1 +1,7 @@
-{}
\ No newline at end of file
+<<<<<<< HACDOCS-672
+{
+    "asciidoc.antora.enableAntoraSupport": true
+}
+=======
+{}
+>>>>>>> main
```

Comment:
I believe you should keep the file unchanged. This is not valid JSON, and the project does not require Antora support.